apps/nrm/models/submission.js

```diff
@@ -0,0 +1,183 @@
+'use strict';
+
+/* eslint complexity: 0 max-statements: 0 */
+
+module.exports = data => {
+  const response = {};
+
+  response.Type = '560734';
+  response['Customer.FirstName'] = data['pv-name-first-name'];
+  response['Customer.Surname'] = data['pv-name-last-name'];
+  response['Customer.Address'] = data['pv-contact-details-street'];
+  response['Customer.Town'] = data['pv-contact-details-town'];
+  response['Customer.County'] = data['pv-contact-details-county'];
+  response['Customer.Postcode'] = data['pv-contact-details-postcode'];
+  response['Customer.ContactMethod'] = data['pv-contact-details'];
+  response['Customer.Alias'] = data['pv-name-nickname'];
+  if (data['pv-dob']) {
+    response['Customer.DOBKnown'] = 'Yes';
+    response['Customer.DOB'] = data['pv-dob'];
+  }
+  // response['Customer.AgeGroup'];
+  const genderMap = {
+    'female': 'Female',
+    'male': 'Male',
+    'unknown': 'Other'
+  };
+  response['Customer.iGender'] = genderMap[data['pv-gender']];
+  response['Customer.Email'] = data['pv-contact-details-email-input'];
+  response['Customer.Mobile'] = data['pv-phone-number-yes'];
+  response['Customer.Mobile'] = data['pv-phone-number-yes'];
+  // response['Customer.ContactTime'];
```

Comment:
do we need this comment?

Reply:
yes because it's in the API but we don't know what to do with it yet